src/effects/native/balanceeffect.cpp

```diff
@@ -1,7 +1,11 @@
 #include "balanceeffect.h"
 
-namespace {
+#include "util/defs.h"
 
+namespace {
+    double kMaxCorner = 22000; // Hz
```

Comment:
It, sounds different. With the bypass freq you can go from dry to wet and back with filter effect sound, a different transition compared to using the gain knobs.

I think it is a good practice to define named variables for all constants.